src/jit/IntMath64Inl.h

```diff
@@ -474,6 +473,12 @@ static bool emitCompare(sljit_compiler* compiler, Instruction* instr)
                 }
 
                 sljit_emit_op2u(compiler, opcode, params[0].arg, params[0].argw, params[1].arg, params[1].argw);
+
```

Comment:
Similar to before, the `instr->opcode() != ByteCode::I32AndOpcode` is not handled as well.